skeleton/local-deploy/deployment/user-catalog.json

```diff
@@ -0,0 +1,27 @@
+{
+	"recipes": [
```

Comment:
nit: probably best to fix indentation

Reply:
Weird, locally for me it is indented properly. Let me double check if anything is changing it on push or something